sfdx-source/apex-common-samplecode/test/classes/domains/OpportunitiesTest.cls

```diff
@@ -35,8 +35,7 @@ private class OpportunitiesTest
 	{
 		// Create mocks
 		fflib_ApexMocks mocks = new fflib_ApexMocks();
-		fflib_ISObjectUnitOfWork uowMock = new fflib_SObjectMocks.SObjectUnitOfWork(mocks);
-
+		fflib_SObjectUnitOfWork uowMock = (fflib_SObjectUnitOfWork) mocks.mock(fflib_SObjectUnitOfWork.class);
```

Comment:
@ctchipps -- please use the UOW interface (`fflib_ISObjectUnitOfWork`) and not the implementation itself when mocking.

Reply:
Updated to use the interface